language/ui-patterns/patterns/elements/index.md

```diff
@@ -0,0 +1,8 @@
+---
+type: ??
```

Comment:
@Snugug I don't know what content type to add to this. Its doesn't quite fit under "Guidelines" or "Patterns" What do you suggest?